plugins/content_distribution/providers/cross_kaltura/lib/batch/CrossKalturaDistributionEngine.php

```diff
@@ -228,8 +228,9 @@ protected function getEntryObjects(KalturaClient $client, $entryId, KalturaDistr
 	    
 	    // get entry
 	    KalturaLog::debug('Getting entry id ['.$entryId.']');
-	    $entry = $client->baseEntry->get($entryId);
-	    
+		KBatchBase::impersonate($this->distributionProfile->partnerId);
```

Comment:
No problem. The reason I didn't want to do it this way is because there are a few calls I didn't touch like for metadata objects and thumb assets. 
I'll test to verify the change doesn't affect functionality